ecommerce_integrations/controllers/customer.py

```diff
@@ -48,6 +48,13 @@ def sync_customer(self, customer_name: str, customer_group: str) -> None:
 		except Exception as e:
 			raise e
 
+	def get_customer_address_doc(self,customer_name: str, type):
+		doc = frappe.get_all("Address", {"link_name": customer_name, "address_type": type})
+		if(len(doc)>0):
+			doc = frappe.get_last_doc("Address", {'name': doc[0].name})
```

Comment:
Just general comment. Format as per PEP8 / black. 

To automate this, you can use a fork of `black`  that uses tabs instead of spaces: https://github.com/adityahase/black